dcp_prototype/frontend/.env.development

```diff
@@ -6,3 +6,5 @@ AUTH0_CALLBACK=http://localhost:8000/
 
 API_URL=https://u5j1wa9u5i.execute-api.us-east-1.amazonaws.com/dev
 CXG_URL=http://cellxgene-dev.us-west-2.elasticbeanstalk.com
+
+AUDIENCE=https://browser-api.dev.single-cell.czi.technology
```

Comment:
I [updated](https://github.com/chanzuckerberg/dcp-prototype/pull/279/files#diff-acd4877c5205d1b80fcb5cef69e55ddcR7) the local frontend to point to the local backend at http://localhost:8001 . If that makes sense to you, this should point there as well

Reply:
The audience wont change in this case. 

I made those same changes to connect the frontend to the backend here https://github.com/chanzuckerberg/dcp-prototype/pull/272. Mine includes the ability to connect the python debugger to your local chalice app. Please take a look and merge into your PR.

Reply:
Oh sorry I missed this. Looks great - we'll merge yours first and I'll update my PR to reflect it